packages/bitcoin-dlc-provider/lib/BitcoinDlcProvider.ts

```diff
@@ -1417,6 +1435,79 @@ Payout Group not found',
     return sigs.flat();
   }
 
+  async VerifySignatures(
+    _dlcOffer: DlcOffer,
+    _dlcAccept: DlcAccept,
+    _dlcTxs: DlcTransactions,
+    _dlcCloses: DlcClose[],
+    rawCloseTxs: string[],
+    isOfferer: boolean,
+  ): Promise<boolean> {
+    const { dlcOffer, dlcAccept, dlcTxs } = checkTypes({
+      _dlcOffer,
+      _dlcAccept,
+      _dlcTxs,
+    });
+
+    const dlcCloses = _dlcCloses.map(
+      (_dlcClose) => checkTypes({ _dlcClose }).dlcClose,
+    );
+
+    const network = await this.getConnectedNetwork();
+
+    const fundingPubKeys =
+      Buffer.compare(dlcOffer.fundingPubKey, dlcAccept.fundingPubKey) === -1
+        ? [dlcOffer.fundingPubKey, dlcAccept.fundingPubKey]
+        : [dlcAccept.fundingPubKey, dlcOffer.fundingPubKey];
+
+    const p2ms = payments.p2ms({
+      m: 2,
+      pubkeys: fundingPubKeys,
+      network,
+    });
+
+    const paymentVariant = payments.p2wsh({
+      redeem: p2ms,
+      network,
+    });
+
+    const pubkey = isOfferer ? dlcAccept.fundingPubKey : dlcOffer.fundingPubKey;
+
+    const sigsValidity: Promise<boolean>[] = [];
+
+    for (let i = 0; i < rawCloseTxs.length; i++) {
+      const rawTx = rawCloseTxs[i];
+      const dlcClose = dlcCloses[i];
+
+      const verifySignatureRequest: VerifySignatureRequest = {
+        tx: rawTx,
```

Comment:
Looks like this only checks signatures. How do we verify whether the `offerPayoutSatoshis` and `acceptPayoutSatoshis` matches the transaction being signed for?

Reply:
offerPayoutSatoshis and acceptPayoutSatoshis are already verified since rawTx includes these outputs